array/last.js

```diff
@@ -0,0 +1,12 @@
+if (!Array.prototype.last) {
+  Object.defineProperty(Array.prototype, 'last', {
+    value: function(num) {
+      if(num){
```

Comment:
instead of an if statement and reusing this, why not use the ternary operator from array/first.js?